pom.xml

```diff
@@ -60,7 +60,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.4.3</version>
+                <version>3.1.0</version>
```

Comment:
Make sure this works. I remember trying 3.0 and hitting some bug that blocked upgrading